hhs_oauth_server/request_logging.py

```diff
@@ -86,12 +86,21 @@ def __str__(self):
             log_msg['access_token_hash'] = hashlib.sha256(str(access_token).encode('utf-8')).hexdigest()
 
         # Auth flow trace logging.
-        if self.request.session.get('auth_uuid', None):
+        if self.request.session:
             if is_path_part_of_auth_flow_trace(self.request.path):
                 auth_flow_dict = get_session_auth_flow_trace(self.request)
 
                 for k in SESSION_AUTH_FLOW_TRACE_KEYS:
-                    log_msg[k] = auth_flow_dict.get(k, None)
+                    if auth_flow_dict.get(k, None):
+                        log_msg[k] = auth_flow_dict.get(k, None)
+
+        # Get FHIR_ID if available.
+        user = get_user_from_request(self.request)
+        if user:
+            log_msg['user'] = str(user)
+            if getattr(user, 'crosswalk', None):
+                if getattr(user.crosswalk, 'fhir_id', None):
+                    log_msg['fhir_id'] = str(user.crosswalk.fhir_id)
```

Comment:
Just curious if there is a simpler way to do this, like wrapping in a try/catch statement?